packages/apps/plugins/plugin-metadata/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@braneframe/plugin-metadata",
```

Comment:
my current PR envisions the plugin-registry which might handle this also?
https://github.com/dxos/dxos/pull/4532